app/controllers/students_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
   end
 
   def edit
+    redirect_to assignments_path unless Time.now <= $studio_due
     @student = Student.find(params[:id])
```

Comment:
Can we also present some message to the user after the page redirect, you can add a `flash[:notice] = "Studio signup is due."`